ethereum/contracts/ETHApp.sol

```diff
@@ -50,11 +51,15 @@ contract ETHApp is RewardController, AccessControl {
     bytes32 public constant CHANNEL_UPGRADE_ROLE =
         keccak256("CHANNEL_UPGRADE_ROLE");
 
+    ETHVault public vault;
```

Comment:
This storage variable is only ever set once in the constructor so can make it an immutable constant.

Especially since we're designing EthApp to be immutable. Don't think we'll ever need to swap new vaults into an app that has already been deployed.